Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/SessionController.java

```diff
@@ -36,11 +52,16 @@ public boolean createSession(String roomName, String password, String hostName,
             sessions.put(roomName, wrapper);
         }
 
-        GameServer.getInstance().getRmiControler().sessionCreated(roomName, hostName, !password.isEmpty(), capacity);
+        GameServer.getInstance().getRmiController().sessionCreated(roomName, hostName, !password.isEmpty(), capacity);
         LOGGER.log(Level.INFO, "Session {0} created.", new String[]{roomName});
         return true;
     }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)